src/core/service.c

```diff
@@ -1911,7 +1915,8 @@ static void service_enter_stop_post(Service *s, ServiceResult f) {
         if (s->control_command) {
                 s->control_command_id = SERVICE_EXEC_STOP_POST;
 
-                r = service_spawn(s,
+                r = service_spawn(__func__,
```

Comment:
maybe instead rename service_spawn() to service_spawn_internal() and then just do:

```c
#define service_spawn(...) service_spawn_internal(__func__, __VA_ARGS__)
```

that would make the change a lot more minimal, and add less noise to the invocations.